package.json

```diff
@@ -5,20 +5,20 @@
     "glob": "^7.1.7"
   },
   "devDependencies": {
-    "@codecademy/eslint-config": "^8.0.0",
-    "@codecademy/prettier-config": "^0.1.10",
-    "@codecademy/tsconfig": "^0.2.0",
-    "@types/glob": "^7.1.4",
-    "@types/jest": "^26.0.24",
-    "@types/node": "^16.4.3",
-    "eslint": "^7.25.0",
-    "husky": "^7.0.4",
-    "jest": "^27.0.6",
-    "lint-staged": "^12.1.2",
-    "markdownlint-cli": "~0.28.1",
-    "prettier": "^2.2.1",
-    "ts-jest": "^27.0.4",
-    "typescript": "^4.2.4"
+    "@codecademy/eslint-config": "^6.0.0",
```

Comment:
I had to downgrade this to match the `eslint` version and missing `@typescript-eslint/eslint-plugin`